docs/cloud/serverless.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Serverless Preview 
+layout: default
+parent: Cloud
+has_children: false
+nav_order: 14 
+---
+
+# FeatureBase Serverless
+
+{: .no_toc }
+
+{% include /serverless/serverless-preview-warning.md %}
+
+FeatureBase Serverless is a new architecture for FeatureBase that separates compute power and storage and allows each to scale independently.
+
+{% include page-toc.md %}
+
+## Why Serverless?
```

Comment:
Asking a "Why" frames the information that follows in context of sales and marketing.

Help presents important information for the user, it's not S&M :)

So this heading can be removed.